RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -115,16 +115,21 @@ import Foundation
         }
     }
 
-    /// Register user preference to the IAM SDK.
-    /// - Parameter preference: Preferences of the user.
-    @objc public static func registerPreference(_ preference: IAMPreference?) {
+    /// Register user preference object to the IAM SDK.
+    ///
+    /// Registered object should be updated to reflect current user session state.
+    /// There is no need to call this method again, unless new `UserInfoProvider` object has been created.
```

Comment:
Maybe this is better:

There is no need to call this method again -> Should only be called once